route-rs-afpacket/src/output.rs

```diff
@@ -0,0 +1,100 @@
+use afpacket;
+use futures::{
+    self, ready,
+    task::{Context, Poll},
+    Future, Stream,
+};
+use route_rs_runtime::link;
+use std::pin::Pin;
+
+#[derive(Default)]
+pub struct AfPacketOutputLink {
```

Comment:
Need documentation before we merge. 
Please describe what this link is, and how it is useful.

Reply:
Nit: Remove Link from the name,  accidental Hungarian.